samples/python/vlm_chat_sample/vlm_chat_sample.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+
+import numpy as np
+import openvino_genai
+from PIL import Image
+from openvino import Tensor
+
+
+def streamer(subword):
+    print(subword, end='', flush=True)
+    # Return flag corresponds whether generation should be stopped.
+    # False means continue generation.
+    return False
+
+
+def read_image(path):
+    pic = Image.open(path)
+    image_data = np.array(pic.getdata()).reshape(1, 3, pic.size[1], pic.size[0]).astype(np.byte)
+    return Tensor(image_data)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('model_dir')
+    parser.add_argument('image_dir')
+    args = parser.parse_args()
+
+    image = read_image(args.image_dir)
+
+    device = 'CPU'  # GPU can be used as well
+    pipe = openvino_genai.VLMPipeline(args.model_dir, device)
+
+    config = openvino_genai.GenerationConfig()
+    config.max_new_tokens = 100
+
+    pipe.start_chat()
+    while True:
+        try:
+            prompt = input('question:\n')
+        except EOFError:
+            break
+        pipe(prompt, [image], config, streamer)
```

Comment:
kwargs approach is expected to be the main one. Can you move to it

Reply:
Done.